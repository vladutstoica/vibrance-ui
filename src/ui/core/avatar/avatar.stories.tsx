import type { Meta, StoryObj } from "@storybook/react";

import React from "react";
import { Avatar, AvatarFallback, AvatarImage } from "./avatar";

/**
 * An image element with a fallback for representing the user.
 */
const meta = {
  title: "Design System/Core/Avatar",
  component: Avatar,
  tags: ["autodocs"],
  argTypes: {},
  render: (args) => (
    <Avatar {...args}>
      <AvatarImage src="https://placehold.co/1" alt="Avatar" />
      <AvatarFallback>CN</AvatarFallback>
    </Avatar>
  ),
  parameters: {
    layout: "centered",
  },
} satisfies Meta<typeof Avatar>;

export default meta;

type Story = StoryObj<typeof meta>;

/**
 * The default form of the avatar.
 */
export const Default: Story = {};
