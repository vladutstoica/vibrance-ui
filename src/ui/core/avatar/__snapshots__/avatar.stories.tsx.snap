// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Design System/Core/Avatar Default smoke-test 1`] = `
<span class="relative flex h-10 w-10 shrink-0 overflow-hidden rounded-full">
  <img class="aspect-square h-full w-full"
       alt="Avatar"
       src="https://placehold.co/1"
  >
</span>
`;
